app/helpers/physical_storage_helper/textual_summary.rb

```diff
@@ -75,7 +75,11 @@ def textual_machine_type
   end
 
   def textual_model
-    {:label => _("Model"), :value => @record.asset_detail&.model}
+    if @record.ext_management_system.kind_of?(ManageIQ::Providers::Autosde::StorageManager)
+      {:label => _("Model"), :value => @record.ext_management_system.physical_storage_families.find(@record.physical_storage_family_id).name}
```

Comment:
I know we don't want to go this route, but some ActiveRecord advice the way you get to the name of the physical_storage_family from a physical_storage is via the `@record.physical_storage_family` association.  This would then be
```suggestion
      {:label => _("Model"), :value => @record.physical_storage_family.name}
```

By manually doing the query you bypass any caching rails might have done and force an extra query that might not be required

Reply:
ok. fixed that.